.circleci/config.yml

```diff
@@ -284,6 +284,24 @@ defaults:
             name: Install build dependencies
             command: ./.circleci/osx_install_dependencies.sh
 
+  - steps_install_python_dependencies: &steps_install_python_dependencies
```

Comment:
I'd make it a command like `gitter_notify_unless_pr` or `prepare_bytecode_report`.

The current step templates should be converted to that too actually. They always replace the whole step list, which is inconvenient if you need to add one more step in your job. That's why we use that weird `when` block with an always true condition as a workaround. Commands don't have this problem.

Both this and adding the new commands could do into a separate, smaller PR.

Reply:
I made it a command, but just realized now that you said to do it in a separated PR, sorry. I will split it in another PR.

Reply:
Done in: https://github.com/ethereum/solidity/pull/14140